libs/etterlatte-behandling-model/src/test/kotlin/no/nav/etterlatte/libs/common/behandling/InnstillingTilKabalTest.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.etterlatte.libs.common.behandling
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.databind.json.JsonMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+
+class InnstillingTilKabalTest {
+    private val objectMapper: ObjectMapper =
+        JsonMapper.builder()
+            .addModule(JavaTimeModule())
+            .addModule(KotlinModule.Builder().build())
+            .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+            .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+            .enable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES)
+            .enable(DeserializationFeature.FAIL_ON_NUMBERS_FOR_ENUMS)
+            .enable(JsonGenerator.Feature.WRITE_BIGDECIMAL_AS_PLAIN)
+            .build()
+
+    @Test
+    fun testMedTekst() {
+        val innstillingTilKabal =
+            objectMapper.readValue<InnstillingTilKabal>(
+                """
+            {
+                "lovhjemmel": "FTRL_1_3_A",
+                "brev": {"brevId": 42},
+                "tekst": "b"
+            }""",
+            )
+        innstillingTilKabal.lovhjemmel shouldBe KabalHjemmel.FTRL_1_3_A
+        innstillingTilKabal.internKommentar shouldBe "b"
+    }
+
+    @Test
+    fun testMedInternKommentar() {
+        val innstillingTilKabal =
+            objectMapper.readValue<InnstillingTilKabal>(
+                """
```

Comment:
Her burde du lage objektet direkte

Reply:
Dette er for å teste deserialisering. Jeg støtter at "tekst" fortsatt kan finnes i JSON-objektet i DB. Men hvis ingen oppdaterer Klage-tabellen med gammel kode akkurat mens deploy pågår så trengs ikke dette, siden flyway skriver om alle radene :) Bør jeg droppe det og heller bare sjekke etter deploy at ingen gamle"tekst"-felter henger igjen?

Reply:
Tror ikke du trenger dette, siden flyway går igjennom migreringene du har definert når du starter opp appen, og da har du også med ny kode

Reply:
Hmm ja lag objektet.toJson også gjør det du skal så brekker det om man gjør endringer i InnstillingTilKabal på en mer fornuftig måte